params.jinja

```diff
@@ -55,7 +55,7 @@
 {% set python3 =  salt['pillar.get']('python3', False) %}
 {#{% set python3 = python3 if '2017' not in salt_version else False %}#}
 {% set python_major_version = '3' if python3 else '2' %}
-{% set python_version = 'python' %}
+{% set python_version = 'python3' %}
```

Comment:
What about python2? Why are we changing it to python3?

Reply:
Should only be python, changed it.